core/tests/protractor_utils/ExplorationEditorFeedbackTab.js

```diff
@@ -73,7 +77,7 @@ var ExplorationEditorFeedbackTab = function() {
     expect(feedbackTabRow.isPresent()).toBe(true);
   };
 
-  this.getSuggestionThreads = function() {
+  this.getFirstSuggestionThread = function() {
```

Comment:
@rakshitkumarcse, why it's named this way? I see it returns a list of threads, right?

Reply:
Yes. It returns a list. Changing it. Thanks for pointing out?

Reply:
@DubeySandeep Thanks for pointing out. It was unintentional.

Reply:
done